lib/auth/init.go

```diff
@@ -136,36 +136,17 @@ func Init(cfg InitConfig, seedConfig bool) (*AuthServer, *Identity, error) {
 	// and this is NOT the first time teleport starts on this machine
 	skipConfig := seedConfig && !firstStart
 
-	// set the cluster auth prerference, the first time read them from the config
+	// upon first start, set the cluster auth prerference from the configuration file
 	// and create a resource on the backend, after that always read from the backend
-	if !skipConfig {
-		log.Infof("Initializing Cluster Authentication Preference: %+v", cfg.AuthPreference)
+	if firstStart {
```

Comment:
do you only need to do this on first start (with empty db) what about upgrade scenario?